src/mixins/filing-template-mixin.ts

```diff
@@ -868,7 +872,8 @@ export default class FilingTemplateMixin extends DateMixin {
       }
     ))
 
-    if (filing.restoration.relationships) {
+    // store relationships
+    if (filing.restoration.relationships?.length > 0) {
       this.setRestorationRelationships(filing.restoration.relationships)
     }
```

Comment:
Without this check, an empty array was being saved for restoration extensions, but I guess the API didn't care.

It works find with "no" relationships for a restoration extension -- I tested locally.

Reply:
I was looking at other previously filed limited restoration extensions, there was a relationships empty array.
![empty relationshios](https://user-images.githubusercontent.com/122301442/232613917-63e7ec86-6e20-405e-be92-fcf124250b7b.PNG)

So, by the looks of it, it doesn't matter if it's there or not like you said. Are we removing it since it's cleaner to not have it there?

Reply:
It's not only cleaner, it's incorrect to have a relationships array for a restoration extension since we don't prompt for it in the UI (it's only applicable to full restorations or restoration conversions).

Reply:
That makes a lot of sense. Alright, thanks Sev.